### PR TITLE
Allow to disable packages in manager config

### PR DESCRIPTION
**Pull request by aschempp**
complements https://github.com/contao/manager-plugin/pull/13

### CODE REVIEW

**Review by Toflar — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "php": "^7.1",
         "contao/core-bundle": "4.5.*",
         "contao/installation-bundle": "4.5.*",
-        "contao/manager-plugin": "^2.2",
+        "contao/manager-plugin": "dev-feature/plugin-cache as 2.3.9999",
```
Comment (leofeyer) [inline composer.json:16]: Before we merge this, we should merge contao/manager-plugin#13 and change this to `^2.3`.

**Review by leofeyer — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (aschempp) [reply composer.json:16]: Yes, and release 2.3 version of `contao/manager-plugin`

**Comment by aschempp** [comment]
I just realized that theses changes should probably go into Contao 4.5 and not 4.6. First of all, Composer would automatically update `contao/manager-plugin` to v2.3 and we need to adjust the `ContaoKernel` or people will get a deprecation warning forever. Secondly, setting a manager config was already supported in Contao 4.5, there was just no way to disable packages in the plugin, but now there is and we're just migrating to a new dependency.

@contao/developers let me know your thoughts.

**Comment by leofeyer** [comment]
> Composer would automatically update `contao/manager-plugin` to v2.3

Good point. We should probably release this as version 3.0 instead of 2.3 if it is incompatible with existing installations.

> setting a manager config was already supported in Contao 4.5

Doesn't the problem also affect Contao 4.4?

**Comment by aschempp** [comment]
> We should probably release this as version 3.0 instead of 2.3 if it is incompatible with existing installations.

Very much not so, because otherwise all extensions would no longer work (assuming Contao 4.6 only allows plugin 3.0).

> Doesn't the problem also affect Contao 4.4?

Well, the plugin only supports PHP7, so it will not be installed in 4.4 on PHP 5.6. But it might on later PHP versions, so we should probably add a require < 2.3 in 4.4

**Comment by leofeyer** [comment]
> so we should probably add a require < 2.3 in 4.4

This would not work, because we can only change the version constraint in Contao 4.4.21. All prior versions would still be installable with the plugin in version 2.3.

So if at all, the plugin in version 2.3 would have to add a conflict with Contao 4.4.

**Comment by aschempp** [comment]
> This would not work, because we can only change the version constraint in Contao 4.4.21. All prior versions would still be installable with the plugin in version 2.3.

That's fine, they just get the deprecation warning, but everything will work fine. That sounds ok to me.

**Comment by aschempp** [comment]
Closed in favor of #74

**Pull request closed without merging**